raiden/network/transport/matrix.py

```diff
@@ -966,8 +988,17 @@ def _send_raw(self, receiver_address: Address, data: str):
         with self._getroom_lock:
             room = self._get_room_for_address(receiver_address)
         if not room:
+            self.log.error(
+                'No room for receiver',
+                receiver=to_normalized_address(receiver_address),
+            )
             return
-        self.log.debug('Send', receiver=pex(receiver_address), room=room, data=data)
+        self.log.debug(
+            'Send raw',
```

Comment:
Could you please remove: https://github.com/raiden-network/raiden/blob/45c4c47ec08e1e232b5f0b24386c8fc0a46f647a/raiden/network/transport/matrix.py#L250?

With this log line you're updating, the one i am referring to is redundant.